src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,25 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  // Визначте класи для елемента ul
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)